kevlar/tests/test_seqio.py

```diff
@@ -99,7 +99,7 @@ def test_aug_fastq_reader_e2():
 def test_partition_reader_simple():
     infile = kevlar.tests.data_file('part-reads-simple.fa')
     readstream = kevlar.parse_augmented_fastx(kevlar.open(infile, 'r'))
-    partitions = [p for p in kevlar.parse_partitioned_reads(readstream)]
+    partitions = list(kevlar.parse_partitioned_reads(readstream))
```

Comment:
The list invocation of the generate here is more concise than the list comprehension.